src/api/route-services/projects.js

```diff
@@ -118,16 +114,24 @@ class ProjectsService {
       projectIds = projectIds.concat(newProjectIds);
     }
 
-    return this.getProjectsFromIds(new Set(projectIds));
+    // Remove duplicates (when we support multi experiment projects
+    // we might have repeated projectId's)
+    projectIds = [...new Set(projectIds)];
```

Comment:
why making it to list now instead of leaving it to be a set like before?

Reply:
wouldn't making it a list afterwards add more complexity?

Reply:
because I don't see the reason why `getProjectsFromIds` should be receiving a set, it is working on it as if it were an array.

The only reason I can think of that we were making it into a set was to remove duplicates, so I think this it more explicit about that and doesn't force other functions to adapt to a set when they don't need to.

Reply:
doing `[...new Set(projectIds)]` is a fairly common pattern to remove duplicates in javascript: https://stackoverflow.com/questions/9229645/remove-duplicate-values-from-js-array

Reply:
Ok, I see. My original question was about why do we have to make them into a set originally, but now I understand (remove duplicate projectIds). I think that if `getProjectsFromIds` works with Set, I wouldn't be making it into an Array, I wasn't questioning that. Making it to Array adds one more operation, and I don't see the benefits of it. I would revert back to the previous code

Reply:
Resolved in a call